### PR TITLE
Bugfix Miscellaneous and Update JavaDoc

### LINKED ISSUE #172: [PE-D] Feature Flaw
All types of invalid input gave me one single type of error message. It would be better to distinguish which part the user didn't input and give them a friendly reminder. Or at least, print the correct format again when the user always input incorrectly. It seems to be more user friendly.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: Cocokkkk/ped#10

### LINKED ISSUE #174: [PE-D] Functionality Bugs
Adding cake in different category but the error message said different location and exp date.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Cocokkkk/ped#12

### LINKED ISSUE #187: [PE-D] Unexpected termination
`Ctrl + C` exits the program without any error message.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: tzexern/ped#13

### PR DESCRIPTION
**Pull request by SimJJ96**

### CODE REVIEW

File: src/test/java/seedu/fridgefriend/command/HistoryCommandTest.java [hunk 1]
```diff
@@ -18,13 +18,8 @@
     private Fridge fridge;
 
     @BeforeEach
-    public void setUp() {
+    public void setUp() throws InvalidDateException, InvalidQuantityException, RepetitiveFoodIdentifierException {
```
Comment (Leeyp) [inline src/test/java/seedu/fridgefriend/command/HistoryCommandTest.java:21]: Do be slightly careful that order matters in the History command, but seems like the checks pass, so looks good!

**Review by Leeyp — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -262,7 +271,7 @@ You do not have oyster in your fridge.
 
 ### List expiring foods: `expiring`
 
-Displays a list of food items that are expiring within a week.
+Displays a list of food items that has expired and are expiring within a week.
```
Comment (Leeyp) [inline docs/UserGuide.md]: should be 'or' instead of 'and'
Displays a list of food items that are expiring within a week or have already expired.

**Review by Leeyp — APPROVED** [review]
LGTM

**Review by kwokyto — APPROVED** [review]
LGTM

**Review by Vinci-Hu — APPROVED** [review]
LGTM

**Pull request merged**